plugins/aep-consent-installed/plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "category": "AEP Edge Consent",
+  "displayName": "Consent extension was registered",
+  "description": "Verifies the Consent SDK Extension has been installed in the app.",
```

Comment:
Title: Consent SDK extension was registered
Tooltip: Verifies the Consent SDK extension was installed and registered in the client-side app.
Description:
	1. Registered: Consent SDK extension is successfully installed and registered with the Mobile Core.
	2. Not registered - Does it have default collect consent status?
			i. Yes: Default collect consent settings were detected, but the Consent extension is either not installed or not registered with the Mobile Core. Please follow the link for instructions.  Doc link
			ii. No: Consent extension is not installed or registered with the Mobile Core. By default the collect consent settings used for Edge Network events is yes. If you intended to use the Consent extension to control these settings, please follow the steps in the link then reload this validator. https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network/